index.js

```diff
@@ -1,93 +1,81 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
+const path = require('path'),
+  express = require('express'),
+  bodyParser = require('body-parser'),
+  session = require('express-session'),
+  cors = require('cors'),
+  errorhandler = require('errorhandler'),
+  debug = require('debug')('index');
+
+const isProduction = process.env.NODE_ENV === 'production';
 
 // Create global app object
 const app = express();
 
 app.use(cors());
 
 // Normal express config defaults
-app.use(require("morgan")("dev"));
+app.use(require('morgan')('dev'));
+
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
+app.use(require('method-override')());
+
+app.use(express.static(path.join(__dirname, '/public')));
 
 app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
+  session({
+    secret: 'authorshaven',
```

Comment:
What is a secret doing here? What happened to environment variable?

Reply:
I didn't touch that part of the code, that was how it was, but I can take it to the environment variable